src/MICore/LaunchOptions.cs

```diff
@@ -433,7 +435,6 @@ public static XmlReader OpenXml(string content)
             settings.IgnoreProcessingInstructions = true;
             settings.IgnoreWhitespace = true;
             settings.NameTable = new NameTable();
-            settings.XmlResolver = null;
```

Comment:
That is really unfortunate. I guess the best solution then is to invoke the property via reflection (or 'dynamic') if it exists, and otherwise ignore it. Would you mind doing that?